client/me/security-checkup/index.jsx

```diff
@@ -0,0 +1,417 @@
+					},
+					components: {
+						strong: <strong />,
+					},
+				}
+			);
+		}
+
+		return (
+			<SecurityCheckupNavigationItem
+				path={ '/me/account' }
+				materialIcon={ icon }
+				text={ translate( 'Account Email' ) }
+				description={ description }
+			/>
+		);
+	}
+
+	renderRecoveryEmail() {
+		const {
+			accountRecoveryEmail,
+			accountRecoveryEmailActionInProgress,
+			accountRecoveryEmailValidated,
+			translate,
+		} = this.props;
+
+		if ( accountRecoveryEmailActionInProgress ) {
+			return this.renderPlaceholderNavItem();
+		}
+
+		let icon, description;
+
+		if ( ! accountRecoveryEmail ) {
+			icon = this.getWarningIcon();
+			description = translate( 'You do not have a recovery email address.' );
+		} else if ( ! accountRecoveryEmailValidated ) {
+			icon = this.getWarningIcon();
+			description = translate(
+				'You still need to validate your recovery email address: {{strong}}%(recoveryEmailAddress)s{{/strong}}',
+				{
+					args: {
+						recoveryEmailAddress: accountRecoveryEmail,
+					},
+					components: {
+						strong: <strong />,
+					},
+				}
+			);
+		} else {
+			icon = this.getOKIcon();
+			description = translate(
+				'You have set {{strong}}%(recoveryEmailAddress)s{{/strong}} as your recovery email address.',
+				{
+					args: {
+						recoveryEmailAddress: accountRecoveryEmail,
+					},
+					components: {
+						strong: '<strong>',
+					},
+				}
+			);
+		}
+
+		return (
+			<SecurityCheckupNavigationItem
+				path={ '/me/security/account-recovery' }
+				materialIcon={ icon }
+				text={ translate( 'Recovery Email' ) }
+				description={ description }
+			/>
+		);
+	}
+
+	renderRecoveryPhone() {
+		const {
+			accountRecoveryPhone,
+			accountRecoveryPhoneActionInProgress,
+			accountRecoveryPhoneValidated,
+			translate,
+		} = this.props;
+
+		if ( accountRecoveryPhoneActionInProgress ) {
+			return this.renderPlaceholderNavItem();
+		}
+
+		let icon, description;
+
+		if ( ! accountRecoveryPhone ) {
+			icon = this.getWarningIcon();
+			description = translate( 'You do not have a recovery SMS number.' );
+		} else if ( ! accountRecoveryPhoneValidated ) {
+			icon = this.getWarningIcon();
+			description = translate(
+				'You still need to validate your recovery SMS number: {{strong}}%(recoveryPhoneNumber)s{{/strong}}',
+				{
+					args: {
+						recoveryPhoneNumber: accountRecoveryPhone.numberFull,
+					},
+					components: {
+						strong: <strong />,
+					},
+				}
+			);
+		} else {
+			icon = this.getOKIcon();
+			description = translate(
+				'You have set {{strong}}%(recoveryPhoneNumber)s{{/strong}} as your recovery SMS number.',
+				{
+					args: {
+						recoveryPhoneNumber: accountRecoveryPhone,
+					},
+					components: {
+						strong: <strong />,
+					},
+				}
+			);
+		}
+
+		return (
+			<SecurityCheckupNavigationItem
+				path={ '/me/security/account-recovery' }
+				materialIcon={ icon }
+				text={ translate( 'Recovery SMS Number' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **42** times:
`translate( 'Recovery SMS number' )` *ES Score: 15*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/security-checkup)